index.html

```diff
@@ -0,0 +1,105 @@
+  <main>
+    <header class="header">
+      <img class="menu" src="images/icons/menu.svg" alt="" />
+      <div class="nav-menu">
+        <ul class="menu-list">
+          <li class="mobile-nav"><a href="index.html">Home </a> </li>
+          <li class="mobile-nav"> <a href="about.html"> About </a></li>
+        </ul>
+        <div class="close">Close</div>
+      </div>
+      <div class="desktop-bar">
+      </div>
+      <div class="content-nav">
+        <p class="hint">"If everyone is thinking alike, then no one is thinking."</p>
+        <a href="#" class="logo">
+          <h1>The Power within us 2022</h1>
+        </a>
+        <p class="description">
+         This program's sole purpose has always been to create a safe space for developers
+         worldwide. <br>
+         In the ten years of our existence, we've grown as a community through peer support.
+         This year, we hope to help everyone grow as individuals.
+        </p>
+        <p class="date">2022.04.15(FRI) ~ 16(SAT)</p>
+        <p class="location">@ Bells university of Technology</p>
+      </div>
+    </header>
+    <section class="main-program">
+      <h3 class="section-title">Main Program</h3>
+      <hr class="section-line" />
+      <div class="program-content-nav">
+        <div class="program-content">
+          <img src="images/icons/ignite.png" alt="">
+          <p class="content-title">Ignite</p>
+          <p class="content-description">
+            Get familiar with the latest and best coding practices around.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/lecture.png" alt="" />
+          <p class="content-title">Lecture</p>
+          <p class="content-description">
+            Chance to listen to the top developers from all over the world.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/workshop.png" alt="" />
+          <p class="content-title">Workshop</p>
+          <p class="content-description">
+            Contribute via pair programming with programmers from all walks of life.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/exhibition.png" alt="" />
+          <p class="content-title">Contest</p>
+          <p class="content-description">
+            Go head to head against the top and best developers and fancy your chances of winning.
+          </p>
+        </div>
+        <div class="program-content">
+          <img src="images/icons/forum.png" alt="">
+          <p class="content-title">Discuss</p>
+          <p class="content-description">
+            Questions and Answers session.
+          </p>
+        </div>
+      </div>
+      <a class="join-us" href=""> Join TTPS 2022</a>
+      <a href="index.html" class="see-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section class="speakers">
+      <h3 class="section-title">Featured Speakers</h3>
```

Comment:
[OPTIONAL]
- I think it would be better if you adjust the sizes of the speaker images to fit that of the original design. The images as of now are stretched. 🦑